src/main/scala/com/databricks/spark/avro/SchemaConverters.scala

```diff
@@ -101,10 +101,10 @@ private object SchemaConverters {
       val newField = fieldsAssembler.name(field.name).`type`()
 
       if (field.nullable) {
-        newField.nullable()
```

Comment:
I mistakenly thought that this was changing the newField object, and there wasn't a test to catch this error.